test/livebook_web/live/hub/edit_live_test.exs

```diff
@@ -161,5 +161,133 @@ defmodule LivebookWeb.Hub.EditLiveTest do
       refute render(element(view, "#hub-secrets-list")) =~ secret.name
       refute secret in Livebook.Hubs.get_secrets(hub)
     end
+
+    test "creates file system", %{conn: conn, hub: hub} do
+      {:ok, view, _html} = live(conn, ~p"/hub/#{hub.id}")
+
+      bypass = Bypass.open()
+      file_system = build_bypass_file_system(bypass)
+
+      attrs = %{file_system: Livebook.FileSystem.dump(file_system)}
+
+      Bypass.expect_once(bypass, "GET", "/", fn conn ->
+        conn
+        |> Plug.Conn.put_resp_content_type("application/xml")
+        |> Plug.Conn.resp(200, """
+        <ListBucketResult>
+          <Name>mybucket</Name>
+        </ListBucketResult>
+        """)
+      end)
```

Comment:
Perhaps worth moving into `expect_s3_listing(bypass)`, since we use it in a couple tests and it's not the test objective :)